### ISSUE TITLE
Random popup message with "This file exceeds the maximum upload size allowed for your account."

### ISSUE THREAD

**Issue by s0m3guy** [xwiki-labs/cryptpad#652]
**Describe the bug**
Sometimes a message pops up saying that the file size is too big for my account. Occasionally it happens so many times that the sheet becomes absolutely unusable as this message pops up nearly everytime I perform some action.

**Where did it happen?**
In a CryptPad sheet

**To Reproduce**
1. Have the sheet open in a Chrome/Firefox tab
2. Don't do anything or browse in other tabs (hence not even increasing the size), or just click somewhere in the sheet and do something like writing, copy/pasting, etc.
3. All of a sudden this message pops up
4. Clicking OK
5. A blank message pops up

**Expected behavior**
None of this

**Screenshots**
https://imgur.com/a/Fx8rEy4

**Browser (please complete the following information):**
 - OS: Windows 10 x64
 - Browser: Latest Chrome/Firefox
 - Extensions installed: uBlock Origin but other than that none that should matter as far as I can tell

**Additional context**
Unfortunately hard to reproduce as it seems to appear randomly. Also the sheets filze size is less than 1 MB.

**Comment by ansuz** [comment]
CryptPad attempts to save major versions of spreadsheets at least once every 100 minor revisions. It does so by exporting them to an encrypted file (which we call a checkpoint) which is then uploaded to the server. If this file exceeds the maximum size that is allowed on the host instance it will be blocked by the server.

When clients find that no such checkpoints have been created within the last 100 revisions they try to create one. It seems that in your case this happened over and over again because it could never succeed.

We've implemented a fix which avoids showing this warning repeatedly within a single session, however, this won't fix the underlying failure to create a new checkpoint. It's unclear why this was failing, because on CryptPad.fr we allow registered (non-premium) users to upload files of up to 25MB. Either there's some bizarre bug that is causing your 1MB spreadsheet to export as a 25+ MB checkpoint, or you're using the spreadsheet editor on another instance with a lower file size limit.

If the problem persists and isl occurring on CryptPad.fr please submit a support ticket with a link to an affected sheet so we can try to reproduce. If it's happening on another instance you may want to ask the admins to increase the maximum file size limit.
